testing/armagetronad/APKBUILD

```diff
@@ -0,0 +1,46 @@
+# Maintainer: Díaz Urbaneja Diego <sodomon2@gmail.com>
+pkgname=armagetronad
+pkgver=0.2.8.3.4
+pkgrel=0
+pkgdesc="A Tron Clone in 3D."
+url="http://armagetronad.net/"
+arch="all"
+license="GPL-1.0-only"
+depends=""
+makedepends="sdl_image-dev sdl_mixer-dev ftgl-dev libxml2-dev"
+subpackages="$pkgname-doc $pkgname-openrc"
+options="!check" #the tests are deactivated because the make check has not been declared
+source="https://launchpad.net/$pkgname/${pkgver%.*.*}/$pkgver/+download/$pkgname-$pkgver.src.tar.gz"
```

Comment:
Please write out $pkgname instead of using the variable name here.

Reply:
I don't get it very well, please be more specific.

Reply:
instead of writing `$pkgname` in source please write `armagetronad`.

Reply:
Something like that?
source="https://launchpad.net/armagetronad/${pkgver%.*.*}/$pkgver/+download
/$pkgname-$pkgver.src.tar.gz"

Reply:
or exchange all $pkgname from source for armagetronad?

Reply:
Exchange all of them